app/javascript/styles/mastodon/components.scss

```diff
@@ -8221,7 +8222,7 @@ noscript {
 
   .search__input {
     border: 1px solid lighten($ui-base-color, 8%);
-    padding: 10px;
+    padding: 10px 38px 10px 10px;
     padding-inline-end: 28px;
```

Comment:
This is the value you want to change.

Reply:
There might be some confusion, as I haven't seen you take this comment into account (maybe you haven't pushed your current branch?)

The value that needs to be changed if you want to change the padding to make more space around the button is `padding-inline-end`. However, I see in your screenshot that it has been ignored. What browser are you using?

Reply:
Gargron did not suggest any code change here but just pointed out that the line to change would not be the `padding` line but the `padding-inline-end` one.

Which version of Firefox are you using? And which version of Mastodon have you seen the issue on? I'm starting to believe you're trying to fix an issue that was recently fixed in #26113!